doc/source/whatsnew/v0.19.1.txt

```diff
@@ -57,3 +57,4 @@ Bug Fixes
 
 - Bug in ``pd.pivot_table`` may raise ``TypeError`` or ``ValueError`` when ``index`` or ``columns``
   is not scalar and ``values`` is not specified (:issue:`14380`)
+- pd.eval('') and df.query('') now show better error message and raise ``ValueError`` (:issue: `13139`)
```

Comment:
more like

correctly raise ValueError on empty input to ``pd.eval()`` and ``df.query()``. This